pypsa-to-iamc

```diff
@@ -0,0 +1,634 @@
+            var[sh]['Final energy consumption|Gases|Hydrogen'] = (var[sh]['Final energy consumption|Industries|Gases|Hydrogen']
+                  +var[sh]['Final energy consumption|Transportation|Gases|Hydrogen'])
+
+                                                                               
+            var[sh]['Final energy consumption|Buildings|Heating|District heating'] = 0.001*ct*(n.loads_t.p.filter(like ='urban central heat').filter(like =country).sum().sum())   
+            var[sh]['Final energy consumption|District heating']=var[sh]['Final energy consumption|Buildings|Heating|District heating']                                                       
+            var[sh]['Final energy consumption|Buildings|Heating'] = 0.001*ct*(n.loads_t.p.filter(like ='residential rural heat').filter(like =country).sum().sum()
+                  +n.loads_t.p.filter(like ='services rural heat').filter(like =country).sum().sum()+n.loads_t.p.filter(like ='residential urban decentral heat').filter(like =country).sum().sum()                                              
+                  +n.loads_t.p.filter(like ='services urban decentral heat').filter(like =country).sum().sum()+ var[sh]['Final energy consumption|District heating'] )                                 
+                                                                               
+                 #add residential/services water/space heating demand if we have them                                                   
+                                                                               
+            industry_demand=pd.read_table('resources/industrial_energy_demand_elec_s370_37m_{}.csv'.format(year),delimiter=',',index_col=0)
+            var[sh]['Final energy consumption|Industries|Coal|Coal products'] = 0.001*ct*industry_demand['coke'].filter(like=country).sum()*(10e5/3)
+            var[sh]['Final energy consumption|Industries|Coal'] = (0.001*ct*industry_demand['coal'].filter(like=country).sum()*(10e5/3)
+                +var[sh]['Final energy consumption|Industries|Coal|Coal products'])
+            var[sh]['Final energy consumption|Industries|Solid bio and waste|Primary solid biomass'] = 0.001*ct*industry_demand['solid biomass'].filter(like=country).sum()*(10e5/3)                                                                                                                                                                                                                                                  
+            var[sh]['Final energy consumption|Industries|Gases|Fossil|Natural Gas'] = 0.001*ct*industry_demand['methane'].filter(like=country).sum()*(10e5/3)
+            var[sh]['Final energy consumption|Industries|Liquids|Fossil'] = 0.001*ct*industry_demand['naphta'].filter(like=country).sum()*(10e5/3)                                                                      
+             
+                #add EU process emissions?  
```

Comment:
is there a field for this in the IAMC format?

Reply:
I suggest you add this to the second list of questions to ask the SENTINEL team
"Emissions|Kyoto gases|Fossil|CO2|Industries" could be including process emissions and energy-related emissions.